Class-7-(Interfaces).md

```diff
@@ -0,0 +1,19 @@
+# Class 7: Interfaces and Polymorphism
+
+## In the prep work this week, the students learned
+1. What polymorphism is.
+1. What interfaces are and the difference between interfaces and abstract classes.
+1. Different interfaces that are a part of Java and how they can be used in our code.
+1. Best practices around unit tests and interfaces.
+
+## Announcements
+1. Check with your course manager for any important announcements.
+
+## Large Group Time (Instructor)
+
+### Class 7 Topics That Require Careful Attention
+1. Polymorphism has a deceptively simple definition, but can be difficult to grasp. Have students share what polymorphism is to them and share examples to drive home this particular pillar of OOP.
+1. Interfaces may not seem to serve much of a purpose to some. Go over some sample code or class diagram and explain why those behaviors were assigned to an interface. To drive it home, share what behaviors might go in an abstract class.
+
+## Small Group Time: Class 6 Studio (TF Notes)
+1. This studio requires use of given interfaces in Java. Encourage students who are confused to checkout the documentation linked in the book for further explanation of how these things work!
```

Comment:
Remind the TAs to check in with each of their students.

Also, point out particular areas in the studio that are likely to present problems and/or common mistakes that students could make (e.g. forgetting in import a library, extending the interface into the wrong class, etc.).

Reply:
Done! Thanks!